### DISCUSSION TITLE
Not Authorized after forward-auth reboot

### DISCUSSION THREAD

**Discussion by jcogilvie** [thomseddon/traefik-forward-auth#278] [Q&A]
I have a flow that's not working as I would expect, and I'm sure it's a misconfiguration on my part.

I'm using Okta for auth, and it works.  I click a tile and get logged into the app that I have behind traefik-forward-auth.  However, if I restart the traefik-forward-auth service (for instance, changing a config property) any live sessions are broken, and I get "Not Authorized."  If I use Incognito mode or clear my cookies, I get prompted to login again and everything works.

What am I doing wrong?

**Comment by teatonic** [comment]
Hi,

I had the same problem twice
1 - I rebooted the service
2 - After the reboot, the log was indicating an access denied message
3 - I compared the secret on client (forward-auth) and server side (keyloak) and there was no difference
4 - On server side (keyloak) I changed the password (regenerate secret on credential tab)
5 - I did a reboot, and it worked fine
